docs/csharp/language-reference/keywords/foreach-in.md

```diff
@@ -41,10 +41,10 @@ Beginning with C# 7.3, if the enumerator's `Current` property returns a [referen
 
 ## See also
 
-[The foreach statement (C# language specification)](../language-specification/index.md)  
-[Using foreach with Arrays](../../programming-guide/arrays/using-foreach-with-arrays.md)  
-[for](for.md)  
-[Iteration Statements](iteration-statements.md)  
-[C# Keywords](index.md)  
-[C# Reference](../index.md)  
-[C# Programming Guide](../../programming-guide/index.md)  
+[The foreach statement (C# language specification)](/dotnet/csharp/language-reference/language-specification/statements#the-foreach-statement)
+[Using foreach with Arrays](../../programming-guide/arrays/using-foreach-with-arrays.md)
+[for](for.md)
+[Iteration Statements](iteration-statements.md)
+[C# Keywords](index.md)
+[C# Reference](../index.md)
+[C# Programming Guide](../../programming-guide/index.md)
```

Comment:
Same comment as above: either to bring back the trailing white spaces or format the **See also** links as a list.

Reply:
We recently standardized on formatting see also as list. See #6923

Reply:
@mairaw thank you for the update; then the list formatting should be added in both documents.